packages/ubuntu_wsl_setup/lib/pages/installation_slides/installation_slides_model.dart

```diff
@@ -44,27 +46,44 @@ class InstallationSlidesModel extends SafeChangeNotifier {
 
   StreamSubscription? _journalSubs;
 
-  late void Function() _onServerUp;
+  void Function()? _onServerUp;
 
   var _logVisible = false;
 
+  AppModel? _appModel;
+  set appModel(AppModel? details) {
```

Comment:
just a thought while driving by, did you consider passing the attributes that are needed here instead of creating a dependency into AppModel?